big_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -1147,10 +1149,10 @@ match_roster(ItemsValid, Items) ->
     ItemsTokens = [ string:tokens(ItemToken, "\t") || ItemToken <- string:tokens(Items, "\n") ],
 
     true = (length(ItemsValid) == length(ItemsTokens)),
-    true = lists:all(fun({Username, Domain, Nick, Group, Sub}) ->
+    true = lists:all(fun({Username, Domain, _Nick, _Group, _Sub}) ->
                     JID = escalus_utils:jid_to_lower(<<Username/binary, "@", Domain/binary >>),
                     lists:any(fun
-                                ([RosterJID, Nick, Sub, "none", Group]) ->
+                                ([RosterJID, __Nick, __Sub, "none", __Group]) ->
```

Comment:
Ultimately, you're right about the intention (the function was initially introduced in https://github.com/esl/ejabberd_tests/commit/09438c5a#diff-217f8e16b78b791f657c1119ea504f64R519), but I think that since https://github.com/esl/ejabberd_tests/commit/95f4794a#diff-217f8e16b78b791f657c1119ea504f64R670 it doesn't match on `Nick`, `Sub`, or `Nick`.